tests/utils/DummyRequestLogger.js

```diff
@@ -0,0 +1,59 @@
+class DummyRequestLogger {
```

Comment:
Can you use the mock logger in `tests/utils/fakeLogger.js`?  I see this tracks the logging with a counter, but not sure how that's used, or if we can combine these two logger mocks into a single utility.

Reply:
done :) thanks